canonical/canonical.com
# WD-5371 - Remove the locate me input on applications

## Done
- Removed the old locate me mark up and JS

## QA

- Open the demo, apply to a job
- See that there is a new country drop-down and no locate me

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5371
